### PR TITLE
Update Configuration Files

### PR DESCRIPTION
**Pull request by Larox**
This PR includes the following changes:

- Add `.env.example` file for `cakeOrPie`
- Add `dotenv`
- Update in readme to use local env variables

### CODE REVIEW

File: cakeOrPie/README.md [hunk 1]
```diff
@@ -9,3 +9,7 @@ If you want to test this setup, you can send the sample request body with your o
 To receive HTTP requests to localhost, you can run [ngrok](https://ngrok.com/) or launch the app with [Docker](https://www.docker.com/).
 
 Remember to complete the prerequisites in the [outline](../outline.md) file and to change the `to`, `from`, and `templateID` if you want to run this app on your own.
+
+Additionaly, to create environments variables only locally, run `cp .env.example .env` and add the variable value with yours.
```
Comment (jefflinwood) [inline cakeOrPie/README.md]: Maybe say:

To set your environment variables when running locally, copy the .env.example file to .env with the `cp .env.example .env` command. Edit the .env file to add your own environment values.

**Comment by jefflinwood** [comment]
Let's also catch the stack traces here with the 500 internal server error. We can have a simple custom error page:

https://expressjs.com/en/guide/error-handling.html

The stack traces will still show up in the Node.js console for debugging.

File: cakeOrPie/README.md (hunk 1 above)
Comment (Larox) [reply cakeOrPie/README.md]: Updated :)

**Comment by Larox** [comment]
> Let's also catch the stack traces here with the 500 internal server error. We can have a simple custom error page:
> 
> https://expressjs.com/en/guide/error-handling.html
> 
> The stack traces will still show up in the Node.js console for debugging.

I'll take this into account. Updated!

**Comment by jefflinwood** [comment]
Thank you!

**Pull request merged**